{
  test.withException(new TestWithExceptionCallback(){
    @Override public void doTest() throws SubscriptionRepairException, SubscriptionUserApiException {
    }
  }
,ErrorCode.SUB_REPAIR_MISSING_AO_DELETE_EVENT);
}
